charts/authelia/templates/_helpers.tpl

```diff
@@ -216,6 +196,134 @@ Returns the common annotations
     {{- end }}
 {{- end -}}
 
+{{/*
+Returns the injector annotations
+*/}}
+{{- define "authelia.annotations.injector" -}}
+    {{- if include "authelia.enabled.injector" . -}}
+    {{- with $vault := .Values.secret.vaultInjector -}}
+vault.hashicorp.com/agent-inject: "true"
+{{- if $vault.role }}
+vault.hashicorp.com/role: {{ default "authelia" $vault.role }}
+{{- end }}
+{{- if $vault.agent.status }}
+vault.hashicorp.com/agent-inject-status: {{ default "update" $vault.agent.status }}
+{{- end }}
+{{- if $vault.agent.configMap }}
+vault.hashicorp.com/agent-configmap: {{ $vault.agent.configMap }}
+{{- end }}
+{{- if $vault.agent.image }}
+vault.hashicorp.com/agent-image: {{ $vault.agent.image }}
+{{- end }}
+{{- if $vault.agent.initFirst }}
+vault.hashicorp.com/agent-init-first: {{ $vault.agent.initFirst }}
+{{- end }}
+{{- if $vault.agent.command }}
+vault.hashicorp.com/agent-inject-command: {{ $vault.agent.command }}
+{{- end }}
+vault.hashicorp.com/agent-inject-volume-path: {{ include "authelia.secret.mountPath" $ }}
```

Comment:
Correct annotation should be 'vault.hashicorp.com/secret-volume-path: {{ $vault.agent.command }}'.

However, I tested it and it didn't work at all. I found, that it worked only if used per each secret. This is terrible duplication, I will report it in the vault repository. Until it will be fixed we need per secret volume-path.

Reply:
reported that issue at https://github.com/hashicorp/vault-k8s/issues/241

Reply:
it has not been an issue, I just had old vault-k8s version. I'll just fix annotation name.